AzureCalling/app/src/main/res/layout/activity_start_call.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/start_activity"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:background="@color/fragment_background"
+    tools:context="com.azure.samples.communication.calling.views.activities.StartCallActivity">
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="Display Name"
+        android:paddingLeft="16dp"
+        android:paddingTop="24dp"
+        android:paddingBottom="8dp"
+        android:background="@color/fragment_background"
+        android:textColor="#6E6E6E"
+        android:textSize="13sp"/>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/fragment_background"
+        android:orientation="horizontal">
+
+        <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:paddingTop="13dp"
+            android:paddingLeft="16dp"
+            android:paddingBottom="13dp"
+            android:background="@color/colorPrimary"
+            android:src="@drawable/ic_shape" />
+
+        <EditText
+            android:theme="@style/Theme.Fluent"
+            android:id="@+id/start_call_display_name"
+            android:layout_width="match_parent"
+            android:layout_height="48dp"
+            android:hint="Enter a name"
+            android:textSize="17sp"
+            android:maxLines="1"
+            android:inputType="text"
+            android:textColor="@color/textbox_primary"
+            android:paddingTop="13dp"
+            android:paddingBottom="13dp"
+            android:paddingLeft="20dp"
+            android:background="@color/colorPrimary"/>
+    </LinearLayout>
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/fragment_background"
+        android:paddingLeft="16dp"
+        android:textSize="12dp"
+        android:text="Name shown to the others on the call" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
```

Comment:
Why are we using layout_weight of 1 and also a specific width and height? Shouldn't one of the attribute (width or height) be 0dp?

Reply:
0dp makes the layout disappears in the smaller screen. wrap content made it visible.